chanzuckerberg/idseq-web
# Show project and viz name in viz list, overwrite on save

# Description

![image](https://user-images.githubusercontent.com/28797/54792836-10ab9480-4bfd-11e9-867d-3bc1a892f5bb.png)

This adds the info to the backend needed for the ultimate design, and it changes the save behavior from append to overwrite.

NOTE: the table render design still needs work. 

# Test and Reproduce
Outline the steps to test or reproduce the PR here.

1. try endpoint, http://localhost:3000/visualizations.json?domain=library
2. see data for both heatmap and phylo tree
3. see correct titles for  heatmap and phylo tree
4. try same in viz list UI
